certbot/main.py

```diff
@@ -388,6 +389,75 @@ def _init_le_client(config, authenticator, installer):
 
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
+def _delete_if_appropriate(config): # pylint: disable=too-many-locals
+    """Does the user want to delete their now-revoked certs?"""
+    display = zope.component.getUtility(interfaces.IDisplay)
+
+    if config.namespace.noninteractive_mode:
```

Comment:
`config.noninteractive_mode` is a more common pattern in the codebase; config just passes through to namespace